tests/e2e/test_rosacli_version.go

```diff
@@ -53,4 +54,47 @@ var _ = Describe("Get CLI version",
 				Expect(stdout).To(ContainSubstring(info.DefaultVersion))
 			},
 		)
+
+		It("list versions can work correctly for hosted-cp cluster via ROSA cli - [id:62088]",
+			labels.Medium,
+			func() {
+				By("Init the client")
+				rosaClient = rosacli.NewClient()
+
+				By("list the versions with --hosted-cp")
+				versionService := rosaClient.Version
+
+				output, err := versionService.ListVersions("", true)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(rosaClient.Parser.TextData.Input(output).Parse().Tip()).
+					Should(ContainSubstring(
+						"INFO: Hosted cluster upgrades are cluster-based. " +
+							"To list available upgrades for a cluster, please use 'rosa list upgrades'"))
+
+				By("list the versions with channel group")
```

Comment:
If it uses ListAndReflectJsonVersions to check the version, how can it ensure the output `rosa list version --hosted-cp` without '-o json' is correct, this command with the "-o" flag is more often used by customers